### PR TITLE
[WEB-2714] Set patient MRN and other details while accepting invite to clinic

### PR DESCRIPTION
**Pull request by clintonium-119**
See [WEB-2714]

Related PR: https://github.com/tidepool-org/platform-client/pull/161

[WEB-2714]: https://tidepool.atlassian.net/browse/WEB-2714?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/redux/actions/async.js [hunk 1]
```diff
@@ -2384,17 +2384,17 @@ export function fetchPatientInvites(api, clinicId) {
  * @param {String} clinicId - Id of the clinic
  * @param {String} inviteId - Id of the invite
  */
-export function acceptPatientInvitation(api, clinicId, inviteId) {
+export function acceptPatientInvitation(api, clinicId, inviteId, patientId, patientDetails) {
```
Comment (krystophv) [inline app/redux/actions/async.js:2387]: Should probably add param details to the docblock.

File: test/unit/pages/share/PatientInvites.test.js [hunk 2]
```diff
@@ -231,7 +238,7 @@ describe('PatientInvites', () => {
       expect(table().find('tr').at(1).text()).contains('Patient Two')
     });
 
-    it('should allow accepting a patient invite', () => {
+    it('should allow accepting a patient invite directly for tier0200 clinics', () => {
```
Comment (krystophv) [inline test/unit/pages/share/PatientInvites.test.js:241]:
```suggestion
    it('should allow accepting a patient invite directly for tier0100 clinics', () => {
```

**Review by krystophv — APPROVED** [review]
Couple suggestions to consider. LGTM 🚀

File: app/redux/actions/async.js (hunk 1 above)
Comment (clintonium-119) [reply app/redux/actions/async.js:2387]: Will do.

**Pull request merged**